scripts/typescript_checks.py

```diff
@@ -228,6 +228,8 @@
     'core/templates/filters/format-timer.pipe.ts',
     'core/templates/filters/parameterize-rule-description.filter.spec.ts',
     'core/templates/filters/parameterize-rule-description.filter.ts',
+    'core/templates/filters/parameterize-rule-description.pipe.spec.ts',
+    'core/templates/filters/parameterize-rule-description.pipe.ts',
```

Comment:
As in previous [PR](https://github.com/oppia/oppia/pull/15604#discussion_r906697089). Interaction Type needs to fix the same error came in these files too in strict mode. fixing the issue in the root file will fix the type issue in all files.